src/react-apps/applications/service-development/src/App.tsx

```diff
@@ -30,204 +38,83 @@ class AppClass extends React.Component<IAppProps, any> {
-                    path={route.path}
-                    exact={route.exact}
-                    render={(props) => <LeftDrawerMenu
-                      {...props}
-                      menuType={route.menu}
-                    />}
+                    exact={true}
+                    path={route.from}
+                    render={() => (
+                      <Redirect to={route.to} />
+                    )}
                   />
                 ))}
-              </div>
-              <div style={{ paddingLeft: 73 }}>
                 {routes.map((route, index) => (
                   <Route
                     key={index}
                     path={route.path}
-                    render={(props) => <route.subapp
+                    exact={route.exact}
+                    render={(props) => <AppBarComponent
                       {...props}
-                      name={route.path}
+                      org={org}
+                      service={service}
+                      showSubHeader={true}
+                      activeSubHeaderSelection={route.activeSubHeaderSelection}
+                      activeLeftMenuSelection={route.activeLeftMenuSelection}
+                      backgroundColor='primary'
                     />}
                   />
                 ))}
-              </div>
-
+              </Grid>
+              <Grid item={true} xs={12}>
+                <Hidden smDown>
+                  <div style={{ top: 50 }}>
+                    {routes.map((route, index) => (
+                      <Route
+                        key={index}
+                        path={route.path}
+                        exact={route.exact}
+                        render={(props) => <LeftDrawerMenu
+                          {...props}
+                          menuType={route.menu}
+                          activeLeftMenuSelection={route.activeLeftMenuSelection}
+                        />}
+                      />
+                    ))}
+                  </div>
+                </Hidden>
+                <div className={classes.subApp}>
+                  {routes.map((route, index) => (
+                    <Route
+                      key={index}
+                      path={route.path}
+                      exact={route.exact}
+                      render={(props) => <route.subapp
+                        {...props}
+                        name={route.path}
+                      />}
+                    />
+                  ))}
+                </div>
+              </Grid>
             </Grid>
-          </Grid>
-        </Router>
+          </Router>
+        </MuiThemeProvider>
       </React.Fragment>
     );
   }
 }
 
-const mapsStateToProps = (
-  state: IServiceDevelopmentAppState,
-): IAppProps => {
-  return {
-    drawerOpen: state.serviceDevelopment.drawerOpen,
-  };
-};
+const mapsStateToProps = (state: IServiceDevelopmentAppState): any => ({});
 
 const App = connect(mapsStateToProps)(AppClass);
```

Comment:
Why connect this Component to the redux-state if you're not using any values from the redux-state?

Reply:
It's default code. We will connect the Component when we're fetching the language.